tests/test_single.py

```diff
@@ -76,6 +76,30 @@ def test_kv_historical(http1_client):
         assert b64decode(res.json()["kvs"][0]["value"]) == f"bar{i}"
 
 
+# pylint: disable=redefined-outer-name
+def test_lease(http1_client):
+    """
+    Test that the sandbox starts.
+    """
+    res = http1_client.lease_grant()
+    check_response(res)
+    lease_id = res.json()["ID"]
+
+    res = http1_client.lease_keep_alive(lease_id)
+    check_response(res)
+
+    res = http1_client.lease_revoke(lease_id)
+    check_response(res)
+
+    res = http1_client.lease_keep_alive(lease_id)
+    logger.info("res: {} {}", res.status_code, res.text)
+    assert res.status_code == 400
```

Comment:
Is this because this isn't yet implemented? If so, a comment here to the GitHub issue would make it clearer.

Reply:
This is 400 because we have revoked the lease, I've added some comments.